rskj-core/src/main/java/org/ethereum/config/SystemProperties.java

```diff
@@ -318,7 +318,7 @@ public String helloPhrase() {
     }
 
     public List<String> peerCapabilities() {
-        return configFromFiles.hasPath("peer.capabilities") ?  configFromFiles.getStringList("peer.capabilities") : new ArrayList<>(Arrays.asList("rsk"));
+        return configFromFiles.hasPath("peer.capabilities") ?  configFromFiles.getStringList("peer.capabilities") : new ArrayList<>(Arrays.asList("rsk","lc"));
```

Comment:
To ask for the presence is a pattern that reoccurs through the code, but it is bad pattern: this should be set in `reference.conf`, that assures that the value can never be null. Besides that, `lc` should **not** be a default

Reply:
We will check that (the pattern).

On the other hand, the Light Client _capability_ **should** be a default. By default any full node should serve Light Clients, removing it will cause that no Full Node will serve them.

I think the root of this problem is the bad way that defaults and config settings are handled in general in the code.